### DISCUSSION TITLE
How to reach GeoServer URL when in a Virtual Machine

### DISCUSSION THREAD

**Discussion by gui13go** [kartoza/docker-geoserver#618] [Q&A]
Hello,

I have been able to use this geoserver image to serve WMS maps, and wrap it in an application with docker-compose.yml. But, when I try to run this same app in an Azure VM, the front-end part cant reach the geoserver service. Do you guys have any idea what that could be? Is it DNS or Network somehow? Thanks in advance.

**Comment by NyakudyaA** [comment]
We need more details about how you have configured it in Azure
